pkg/uki/reset.go

```diff
@@ -55,6 +56,13 @@ func (r *ResetAction) Run() (err error) {
 		}
 	}
 
+	// SelectBootEntry sets the default boot entry to the selected entry
+	err = action.SelectBootEntry(r.cfg, "active")
```

Comment:
This requires the changes to the upgrade/install/reset workflow that @jimmykarily is doing, otherwise this will fail to set the active entry.

But the idea is that after reset is called, the next boot entry should be active always by default. Grub already does this, so systemd-boot needs to follow up the same behavior.

Reply:
changes are in, this comment is now outdated!